src/test/generate-matrix.py

```diff
@@ -196,23 +196,23 @@ def __str__(self):
 
 # Centos7
 matrix.add_build(
-    name="centos7",
-    image="centos7",
+    name="el7",
+    image="el7",
     docker_tag=True,
 )
 
 # Centos8
 matrix.add_build(
-    name="centos8",
-    image="centos8",
+    name="el8",
+    image="el8",
     env=dict(PYTHON_VERSION="3.6", LDFLAGS="-Wl,-z,relro  -Wl,-z,now"),
     docker_tag=True,
 )
 
 # Centos8, system, coverage
 matrix.add_build(
-    name="centos8 - system,coverage",
-    image="centos8",
+    name="el8 - system,coverage",
+    image="el8",
     coverage=True,
```

Comment:
do some of these "Centos" comments need to be updated as well?  May be good to do a grep for it and double check.

Reply:
Yeah, good catch. I had only searched (case sensitively) for `centos`. I'll fix these. Thanks!